### DISCUSSION TITLE
Firefly Importer question

### DISCUSSION THREAD

**Discussion by TheFork001** [firefly-iii/firefly-iii#8424] [Firefly III]
Of course, im runnning firefly import in iis. Firefly is running on https://myweb/firefly and im trying to figure out the whole url portion and how to tie back https://myweb/importer to https://myweb/firefly im confused on that part. can i get some light on that config part in the .env file of the importer?

**Comment by TheFork001** [comment]
is there any other settings like db settings as firefly iii had? I cant even reach the site is why im wondering if i have something wrong configured

**Reply by TheFork001** [reply]
I have actually changed it up and am running them now on their own site, however, The importer could not connect to Firefly III. Please remedy the error below first, and check out the [documentation](https://docs.firefly-iii.org/references/faq/data-importer/general/) if necessary. I have firefly url redirected to https. im confused what else to put on the url settings.

on critical: Error accessing "https://firefly.mysite.com/api/v1/about?". Status code is 401. Body is: {"message":"Unauthenticated","exception":"AuthenticationException"}

**Reply by TheFork001** [reply]
i also don't ever remember getting a callback url any ideas if i missed something?

**Reply by JC5** [reply]
Did you create an access token, or follow the OAuth2.0 flow?

**Reply by TheFork001** [reply]
I did create the access token, am I required to have like authelia or anything like that to have the importer work?

**Reply by JC5** [reply]
Authelia will need an exception configured for Firefly III and the API endpoints. I'll have to look it up. You can search for it in issues and discussions, perhaps you'll find it before I do. I'll put it in the documentation once I find it.

**Reply by TheFork001** [reply]
Right, okay let me tell you how I have it. All i did was change the Firefli-iii url to https://firefly.myweb.com
vanity: https://firefly.myweb.com

Added my firefly access token from the oath and that's it. I don't have any authenticator like authelia at all. Do I need to do anything to the database.php file?

**Reply by JC5** [reply]
Oh no you don't need Authelia and you don't need to change anything. If the token is correct, it should work just fine.

**Reply by TheFork001** [reply]
and to clarify in the image depicted below, it's the top oauth token right? not the personal token. Also, where it asks for a clal back url do i hadd fireflyimporter.myweb.com? or is it something else?

![image](https://github.com/firefly-iii/firefly-iii/assets/92244014/ad27a8a7-525d-47f3-a362-e7ca237cba12)

**Reply by TheFork001** [reply]
I also notice there is no db config in the .en environment. the only place i find that is in the database.php

**Reply by JC5** [reply]
It's the bottom one, not the top one. The data importer needs no database configuration.

**Reply by TheFork001** [reply]
okay weird changing the log to critical fixed my issue. Ill look at the settings. This importer tool is designed for internal use only right? Like if I wanted to use this from another location is it going to ask me to re-authenticate?

**Reply by TheFork001** [reply]
OH I see clear your data and bam it asks to authenticate again!
